index.html

```diff
@@ -57,142 +66,127 @@ <h1 class="domain">My Recent <br class="break"> Works</h1>
 
     <div class="all-card-holders">
         <article class="card-holder">
-            <section>
-                <div class="card">
-                    <div class="empty-box">
+            <div class="card">
+                <div class="empty-box">
 
+                </div>
+                <div class="multi-post-stories">
+                    <h3>Multi-Post Stories <br> Gain + Glory</h3>
+                    <div class="programing-languages">
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                            <li>HTML</li>
```

Comment:
- [ ] I would suggest you to use a section tag for each of the section page , The section tag is similar to the div tag, but it is more meaningful since it wraps logical groups of related content,kindly use the section tag instead of article  for proper semantic reason and easy to read your code.